COPY/usr/bin/generate_miq_server_cert.sh

```diff
@@ -4,5 +4,6 @@ set -e -o pipefail
 CERT="${NEW_CERT_FILE:-/var/www/miq/vmdb/certs/server.cer}"
 KEY="${NEW_KEY_FILE:-$CERT.key}"
 if [ ! -f "$CERT" -a ! -f "$KEY" ]; then
-  (umask 077 ; openssl req -x509 -newkey rsa -days 1095 -keyout $KEY -out $CERT -subj "/CN=server" -nodes -batch)
```

Comment:
NOTE I could just change the umask to `007` because `openssl` creates the `.key` with owner `rw` only so we end up with

```
-rw-rw----. 1 root     manageiq 1107 Aug 18 15:46 server.cer
-rw-------. 1 root     manageiq 1708 Aug 18 15:46 server.cer.key
```

Reply:
we also need to ensure that this file gets the proper selinux privileges as well via `restorecon -R certs`